### DISCUSSION TITLE
THANK YOU

### DISCUSSION THREAD

**Discussion by XGhozt** [Pryaxis/TShock#1783] [General]
Thanks everyone ( @pryaxis ) working on this project, you're amazing! Do you still have a place to accept donations for the project? See you in the Discord. RIP forums.

**Comment by hakusaro** [comment]
@XGhozt nice to see you again!! Currently I'm trying to get individual developers to setup GitHub Sponsors. If you're interested at all in sponsoring someone, you can sponsor individual developers, or you can donate to organizations on https://www.givewell.org/charities/top-charities. We really appreciate the thought!
